stripe/src/main/java/com/stripe/android/StripeResponse.kt

```diff
@@ -46,4 +48,15 @@ internal data class StripeResponse internal constructor(
     override fun toString(): String {
         return "Request-Id: $requestId, Status Code: $responseCode"
     }
+
+    internal fun getHeaderValue(key: String): List<String>? {
```

Comment:
alternatively, i think we can use a Tree Map which supports case-insensitive lookups by default:

```kotlin
val map: TreeMap<String, List<String>> = TreeMap(String.CASE_INSENSITIVE_ORDER)
map.put("FOO", "FOO")

println(map.get("foo")) // FOO
println(map.get("Foo")) // FOO
println(map.get("FOO")) // FOO
```

Reply:
I'll leave as is 😬